CreditCardEntry/src/com/devmarvel/creditcardentry/library/CreditCardForm.java

```diff
@@ -124,7 +126,9 @@ private void init(Context context) {
 		entry.setCardImageView(cardFrontImage);
 		entry.setBackCardImage(cardBackImage);
 		entry.setCardNumberHint(cardNumberHint);
-
+		if (!enabled) {
```

Comment:
CreditCardForm needs a public method to do this via code too. Also will need to be able to enable from code